docs/pages/database-access/guides/aws-cross-account.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: AWS Cross-Account Database Access
+description: How to connect AWS databases in external AWS accounts to Teleport.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="13.0"
+>
+  AWS cross-account database access is available starting from Teleport `13.0`.
+</Details>
+
+You can deploy the Teleport Database Service with AWS IAM credentials in one
+AWS account and use AWS IAM roles to grant Teleport access to databases in
+external AWS accounts.
+
+## Teleport configuration
```

Comment:
Do we thread this headers like configuration steps user  needs to follow. If yes we should add ordering numbers.

Reply:
I don't think so in this case. I put this under "Other Guides" with HA configuration, dynamic registration guide, GUI clients guide - none of them have numbered steps. And I don't think it needs numbered steps because the guide isn't for any specific database type, just AWS in general

Reply:
but I'll defer to the docs team

Reply:
Usually, we put a Prerequisites section in a how-to guide/tutorial where the user is expected to achieve a well-defined result if they meet the prerequisites and follow the steps in the guide.

In this case, I think the guide is more of a conceptual/architecture guide, similar to the High Availability guide, where the starting/ending points are less well defined than in our how-to guides and the emphasis is more on developing an understanding of Teleport's architecture/configuration possibilities.

I would remove the Prerequisites section and make sure the guide is as clear as possible in terms of architecture, configuration settings, etc. since users would be developing more of a conceptual understanding of these things rather than following a tutorial.

I'll take a closer look later tomorrow!

Reply:
I removed the prereqs here: https://github.com/gravitational/teleport/commit/b5a2b4b48d00b5d2a954965c9900f4e50c7c6b48